models/articles.models.js

```diff
@@ -41,7 +42,32 @@ function getArticles() {
     });
 }
 
+function patchArticle(article_id, inc_votes) {
+    if (!/[0-9]+/.test(article_id)) {
+        return Promise.reject({ status: 400, msg: 'invalid article_id' });
```

Comment:
This really _should_ be a PSQL error. I know you've stated that you prefer to give it a custom msg - but you can do that in your error handler, if you state what the err.code is (it'll be on the PSQL error if you console.log it)